src/Workspaces/CSharp/Portable/Utilities/FormattingRangeHelper.cs

```diff
@@ -360,7 +360,9 @@ private static bool IsSpecialContainingNode(SyntaxNode node)
                 node.Kind() == SyntaxKind.FinallyClause ||
                 node.Kind() == SyntaxKind.LabeledStatement ||
                 node.Kind() == SyntaxKind.LockStatement ||
-                node.Kind() == SyntaxKind.FixedStatement;
+                node.Kind() == SyntaxKind.FixedStatement ||
+                node.Kind() == SyntaxKind.GetAccessorDeclaration ||
+                node.Kind() == SyntaxKind.SetAccessorDeclaration;
```

Comment:
It would make sense to also include them. I will add them and write some test cases.